src/libs/UserUtils.ts

```diff
@@ -131,7 +133,7 @@ function isDefaultAvatar(avatarURL?: string): boolean {
  * @param avatarURL - the avatar source from user's personalDetails
  * @param accountID - the accountID of the user
  */
-function getAvatar(avatarURL: string, accountID: number): React.FC<SvgProps> | string {
+function getAvatar(avatarURL: string | React.FC<SvgProps>, accountID: number): React.FC<SvgProps> | string {
```

Comment:
Maybe we need to realign changes with https://github.com/Expensify/App/pull/30328